### ISSUE TITLE
lisp syntax highlighting treats ?\" as the start of a string.

### ISSUE THREAD

**Issue by alexflint** [SublimeTextIssues/DefaultPackages#186]
The lisp syntax highlighting incorrectly treats `?\"` as the start of a string, where in fact it is a character literal. This results in an inversion of string/non-string regions:

![screenshot 2016-04-26 12 11 44](https://cloud.githubusercontent.com/assets/640247/14830941/2014c9d4-0ba8-11e6-8dfb-4200d92029fa.png)

ST3, Build 3103, OSX 10.10

**Comment by wbond** [comment]
I know effectively nothing about lisp, so I looked into this.

It seems `#\` is common lisp and `?\` is emacs lisp? It seems the current syntax supports the former, but not that later. Is that correct?

**Comment by alexflint** [comment]
@wbond Yes that is correct. I should have mentioned that I'm working with elisp.

**Comment by wbond** [comment]
Does syntax highlighting of lisp usually take into account both variants, or are there usually enough differences to cause problems?

**Comment by alexflint** [comment]
I'm actually not sure what other editors do, but there are relatively few syntactic differences between Common lisp / Elisp so it's probably possible to deal with them together. (Most of the differences are at the semantic level.)
